lang/en/qtype_topictagged.php

```diff
@@ -28,10 +28,21 @@
 $string['pluginname_link'] = 'question/type/topictagged';
 $string['pluginnameadding'] = 'Adding a topic tagged question';
 $string['pluginnameediting'] = 'Editing a topictagged question';
-$string['pluginnamesummary'] = 'Select a question from the question bank having the selected difficulty and topic that has not been used in a while';
+$string['pluginnamesummary'] = 'Select a question from the question bank having the selected difficulty and' .
+    'topic that has not been used in a while.<br>' .
+    '<p><font color=#F57542><b>Warning:</b></font><br>' .
+    '<font color=#A14A28>Do not add this question into Question bank. Just add it to a quiz.</font></p><br>' .
+    '<p><font color=#4287F5><b>Note: </b></font>' .
+    '<font color=#1C3C70>Keep in mind that preview shows randomly questions selected by the selected difficulty and topic.</font></p>' .
```

Comment:
```suggestion
    '<font color=#1C3C70>Keep in mind that preview shows random questions selected by the given difficulty and topic, without updating the last_used tag.</font></p>' .
```

Reply:
Do you think the final user (evaluator) should know about our internal representation?
What do you think about:
```suggestion
    '<font color=#1C3C70>Keep in mind that preview shows random questions selected by the given difficulty and topic, without affecting the selection process.</font></p>' .
```

Reply:
You're right, the user should not be given more technical information than needed. This looks much better.